experiment/textcov.py

```diff
@@ -193,8 +273,36 @@ def subtract_covered_lines(self, other: Textcov):
     """Diff another textcov"""
     for function in other.functions.values():
       if function.name in self.functions:
-        self.functions[function.name].subtract_covered_lines(function)
+        self.functions[function.name].subtract_covered_lines(
+            function, self.language)
 
   @property
   def covered_lines(self):
     return sum(f.covered_lines for f in self.functions.values())
+
+  def determine_jvm_arguments_type(self, desc: str) -> List[str]:
+    """Determine list of jvm arguments type"""
```

Comment:
Can you give some more detail on what `desc` is here in the docstring, and what exactly this function is doing?

Reply:
Given.